File: src/codes/Users_Manual/gain/bessel_0.cpp
//~#include<autd3.hpp>
//~int main() {
//~using namespace autd3;
//~const auto x = 0.0;
//~const auto y = 0.0;
//~const auto z = 0.0;
//~const auto nx = 0.0;
//~const auto ny = 0.0;
//~const auto nz = 1.0;
//~const auto theta = 0.0;
Bessel(Point3(x, y, z), Vector3(nx, ny, nz), theta* rad,
       BesselOption{
           .intensity = std::numeric_limits<EmitIntensity>::max(),
           .phase_offset = Phase::zero(),
       });
//~return 0; }

File: src/codes/Users_Manual/gain/bessel_0.cs
~using AUTD3Sharp;
~using AUTD3Sharp.Utils;
~using AUTD3Sharp.Gain;
~using static AUTD3Sharp.Units;
~var x = 0.0f;
~var y = 0.0f;
~var z = 0.0f;
~var nx = 0.0f;
~var ny = 0.0f;
~var nz = 1.0f;
~var theta = 0.0f;
new Bessel(
    pos: new Point3(x, y, z),
    dir: new Vector3(nx, ny, nz),
    theta: theta * rad,
    option: new BesselOption
    {
        Intensity = Intensity.Max,
        PhaseOffset = Phase.Zero
    }
);